packages/manager/modules/pci/src/projects/project/quota/increase-request/increase-request.html

```diff
@@ -0,0 +1,28 @@
+<oui-modal
+    data-heading="{{ 'pci_projects_project_quota_increase_title' | translate }}"
+    data-primary-action="$ctrl.increaseQuota()"
+    data-primary-label="{{:: 'pci_projects_project_quota_increase_submit_label' | translate }}"
+    data-primary-disabled="$ctrl.isLoading"
+    data-secondary-action="$ctrl.goBack()"
+    data-secondary-label="{{:: 'pci_projects_project_quota_increase_cancel_label' | translate }}"
+    data-on-dismiss="$ctrl.goBack()"
+    data-loading="$ctrl.isLoading"
+>
+    <p
+        data-translate="pci_projects_project_quota_increase_content"
+        data-translate-values="{ 'projectId': $ctrl.projectId }"
+    ></p>
+
+    <form novalidate name="textareaForm">
```

Comment:
Maybe you can wrap the `<oui-modal>` component with the `<form>` element.

Because the type `type="submit"` is part of the modal component template.

More details: https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-modal/src/modal.html#L36